costum_events/models/partners.py

```diff
@@ -0,0 +1,6 @@
+from odoo import models, fields, api
+class Partner(models.Model):
+    _inherit = 'res.partner'
+    _rec_name = 'partners'
+    
+    is_orgnizer = fields.Boolean(string='Is Orgnizer') 
```

Comment:
the name should be **is_organization** or **is_customer** so please rename it here and in the XML as will